tests/everflow/everflow_test_utilities.py

```diff
@@ -830,7 +839,7 @@ def get_expected_mirror_packet(mirror_session, setup, duthost, direction, mirror
         expected_packet.set_do_not_care_scapy(packet.IP, "chksum")
         if duthost.facts["asic_type"] == 'marvell':
             expected_packet.set_do_not_care_scapy(packet.IP, "id")
-        if duthost.facts["asic_type"] in ["cisco-8000","innovium"]:
+        if duthost.facts["asic_type"] in ["cisco-8000", "innovium", "broadcom"]:
```

Comment:
Do we need to add the check just for "broadcom-dnx", as we did before ?

Reply:
Fixed. Checking if platform_asic is broadcom-dnx